pipeline/extract/analytics.py

```diff
@@ -0,0 +1,10 @@
+from prefect import task
+
+from pipeline.types import RawAnalytics
+
+
+@task
+def extract_analytics(db) -> RawAnalytics:
```

Comment:
How about we have the `extract_analytics()` task return a `List[AnalyticsEvent]` instead of `RawAnalytics`, which is a dictionary? The dictionary is just how the data is stored in the database, but I think a list of events would be more useful to pipeline code.

We could create a new file `pipeline/types/analytics` and add an `AnalyticsEvent` type to that.

Reply:
Okay I assumed AnalyticsEvent should be defined as a class similar to the frontend. I was wondering what the type of timestamp should be (assuming int for now).

Reply:
It would be nice to convert the timestamp to `datetime.datetime` so that we can do more with it than just a timestamp. Check out the `NotificationInput` type for an example. When you convert the timestamp to a datetime object, you might have to divid by 1000 since it expects seconds but we store milliseconds. There might be an example somewhere else in the codebase.